### ISSUE TITLE
Add Time Zone to Dates and Times

### ISSUE THREAD

**Issue by tadscottsmith** [openmhz/trunk-server#179]
When listening to systems in another time zone, it isn't explicitly noted that all of the calls are shown in local time. I think it would be helpful to include a short time zone abbreviation after each date and time. 

For example, rather than displaying `4:39:18 PM 12/21` it would be helpful to display something like `4:39:18 PM 12/21 EST`.

I can take a look at handling this, but if there are any JS wizards that can handle this in a few seconds, IMHO it would be a nice PR.

**Comment by robotastic** [comment]
I totally agree - I tried to think through this once and got very confused on the best way to handle this. Do I need for people other than set the timezone for the system? So if I am on the east coast and listening to a call on the west coast, the times for the calls should be in PST.

**Comment by tadscottsmith** [comment]
My very simple solution would be to display the time zone from the browser's local time. I think explicitly stating that it is the listener's local time would help them calculate the time offset themselves.

A more complex solution would be to have the system admin enter their local time zone when they configure the system, but that might be difficult for larger state-wide or national systems that span multiple time zones.

![Image](https://github.com/user-attachments/assets/137979d7-abbb-4286-96d4-ef17b0768573)
